modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java

```diff
@@ -0,0 +1,160 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.processor;
+
+import com.liferay.petra.string.StringBundler;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.json.JSONArrayImpl;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.json.JSONUtil;
+import com.liferay.portal.kernel.util.ArrayUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.source.formatter.SourceFormatterArgs;
+import com.liferay.source.formatter.check.UpgradeCatchAllCheck;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+/**
+ * @author Kyle Miho
+ */
+@RunWith(Parameterized.class)
+public class UpgradeCatchAllCheckTest extends BaseSourceProcessorTestCase {
+
+	@Parameterized.Parameters(name = "Testcase-{index}: Testing {0} .{1}")
+	public static Iterable<Object[]> data() throws Exception {
+		List<Object[]> tickets = new ArrayList<>();
+
+		String issueKeysProp = System.getProperty("issue.key", null);
+
+		String[] issueKeys = StringUtil.split(issueKeysProp, StringPool.COMMA);
+
+		ClassLoader classLoader = UpgradeCatchAllCheck.class.getClassLoader();
+
+		JSONArray jsonArray = new JSONArrayImpl(
+			StringUtil.read(
+				classLoader.getResourceAsStream(
+					"dependencies/replacements.json")));
+
+		for (int i = 0; i < jsonArray.length(); i++) {
+			JSONObject jsonObject = jsonArray.getJSONObject(i);
+
+			String issueKey = jsonObject.getString("issueKey");
+
+			if (!ArrayUtil.contains(issueKeys, issueKey) &&
+				(issueKeys.length > 0)) {
```

Comment:
@drewbrokke I cant seperate it into 2 if statements due to our SF standards. I think its fine though since ArrayUtil.contains can never cause an NPE since it returns false if the array is empty. I swapped the order if thats what you meant